sr_multi_moveit/ur10srh_moveit_config/config/joint_limits.yaml

```diff
@@ -7,43 +7,43 @@ joint_limits:
     max_velocity: 3.15
     has_acceleration_limits: false
     max_acceleration: 0
-    min_position: 0.0
-    max_position: 2.5
+#    min_position: 0.0
```

Comment:
Well, there's no point in having some random joint limits. Maybe I should delete them instead of comment out. bu tthis way it is obvious how to use them.